src/app/rundown-view/components/rundown/rundown.component.spec.ts

```diff
@@ -7,8 +7,9 @@ import { DialogService } from '../../../shared/services/dialog.service'
 import { RundownTimingContextStateService } from '../../../core/services/rundown-timing-context-state.service'
 import { PartEntityService } from '../../../core/services/models/part-entity.service'
 import { Logger } from '../../../core/abstractions/logger.service'
+import { RundownEventObserver } from 'src/app/core/services/rundown-event-observer.service'
 
-describe('RundownComponent', () => {
+fdescribe('RundownComponent', () => {
```

Comment:
`fdescribe`?

Reply:
Done, I forgot to remove it after fixing the test.